### ISSUE TITLE
DS4 weird behavior after idle disconnect

### ISSUE THREAD

**Issue by IceLancerSR** [Ryochan7/DS4Windows#1745]
Leave gamepad to idle disconnect (WIFI). Some time later reconnect and attempt to play games.
Big chunk of games will not work. Some recognize gamepad but is not functional , only very few will work.
Retroarch - sees controller but controls are not working
Games - Dragon quest doesn't work either
BigBox from launchbox works.
Games that switch profile to DS4 native mod or emulated ds4 all work.
Those same games do not work if using default profile for xbox.
basically xbox profile stops functioning , controls are not responding.

State persist untill pc is rebooted. Turning off Bluettooth /restarting ds4win nothing works.Only PC restart works.
DS4 v2. Win10 20H2 x64

Problem started happening with instalation of new vigem 1.17.333
Log file attached below
[bug.txt](https://github.com/Ryochan7/DS4Windows/files/5791772/bug.txt)

**Comment by IceLancerSR** [comment]
Addendum https://gamepad-tester.com/ says no gamepad detected
joy.cpl sees xbox gamepad controller and its functional if i go to properties
![bug](https://user-images.githubusercontent.com/4452102/104107948-c9563280-52c0-11eb-9bd0-1b36ce99db50.jpg)

**Comment by IceLancerSR** [comment]
Addendum.. THIS is so freaking weird!
I can't get gamepad to work any more in xbox mode.
Uninstall vigem, restart PC, install vigem again. restart PC.
Start gamepad ok, start ds4win ok. Start gamepad tester http. No gamepad detected???
switch to ds4 mode. Gamepad works. Switch back to xbox , xbox works?? Turn off gamepad (BT OFF) turn on gamepad, xbox doesn't work again.. da hell is happening

**Comment by IceLancerSR** [comment]
I reverted to previous version of vigem and previous version of ds4win untill this is resolved. Gamepad on wifi works normally with it

**Comment by isaakfawkes** [comment]
> ks.
> DS4 v2. Win10 20H2 x64

I had the same thing happen to me yesterday. Resorted to reboot after everything else failed.
